proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,125 +44,73 @@ direction of the policies set by the Board.
-
-  * If the number of Qualified Nominees are equal to or less than the number of TOC seats available to be elected, the Qualified Nominees shall be approved after the nomination period has closed. If there are more Qualified Nominees than open TOC seats available for election by either the the Governing Board, End User TAB or TOC, then that group shall elect the TOC members via a Condorcet vote. A Condorcet vote shall be run using the Condorcet-IRV method through the Cornell online service ([*http://civs.cs.cornell.edu/*](http://civs.cs.cornell.edu/)).
-
-  * If there are fewer Qualified Nominees than open TOC seats available for election by either the Governing Board, End User TAB or TOC, the group shall initiate another round of nominations with each member or individual eligible to nominate submitting at most one (1) nominee.
-
-* Constraints
-
-  *  TOC Members shall serve two-year, staggered terms. The initial six elected TOC members from the Governing Board election shall serve an initial term of three (3) years. The TOC members initially elected by the End User TAB and TOC shall serve an initial term of two (2) years.
-
- * TOC members may be removed by a two-thirds vote of the other TOC members, with the impacted individual ineligible to participate in the vote.
-
- * Any TOC member that misses three (3) consecutive meetings shall be automatically suspended from eligibility to vote until having attended two meetings consecutively. For avoidance of doubt, the suspended TOC member shall be eligible to vote in the second consecutive meeting.
-
- * The TOC charter, model, approach, composition, etc may be modified by a two-thirds vote of the entire Governing Board.
-
-===========
-
+The CPC is in inclusive group with 2 types of members:
+
+* Regular members
+* Voting members
+
+**Regular members** Regular members have the roles and responsibilities
+as outlined below. In any case where it is not specified that a role
+or responsibility applies to a Voting member, then a Regular member is
+implied. Any member of one of the projects under the foundation may request to
+become a regular member by opening a PR to add themselves to the list of
+regular members. Regular members remain for as long as they are active
+within the work of the CPC. Regular members who have not been active
+in GitHub, participated in meetings, or other work of the CPC for 3 months may be
+removed from the list of Regular members. In addition, a Regular CPC member
+can be removed by by voluntary resignation, or by a standard CPC motion.
+
+**Voting members** Voting members have the roles and responsibilities
+as outlined below. Voting members are selected as follows:
+
+* Each top level project many nominate 2 member through a process
+of their choosing. Once nominated the member must be ratified by the
+board before becoming a Voting member.
+
+* 2 Voting member may be nominated by the projects excluding the
```

Comment:
2 Voting *members*

I also propose that we explicitly say non-top-level projects when simply referring to projects on this line.